modular_zzplurt/code/game/preferences.dm

```diff
@@ -0,0 +1,19 @@
+/datum/preference/toggle/cyborg_sleepers
+	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
+	savefile_identifier = PREFERENCE_PLAYER
+	savefile_key = "sleeper_pref"
+	default_value = FALSE
+
+/datum/preference/toggle/erp/vore_digestable
+	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
+	savefile_identifier = PREFERENCE_PLAYER
+	savefile_key = "digestable_pref"
+	default_value = FALSE
+
+
+/datum/preference/toggle/erp/vore_sounds
+	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
+	savefile_identifier = PREFERENCE_PLAYER
+	savefile_key = "vore_sounds_pref"
+	default_value = FALSE
+
```

Comment:
These should be handled by vore prefs

`/datum/preference/toggle/erp/vore_digestable` is replaceable by existing `/datum/vore_pref/toggle/digestion`
`/datum/preference/toggle/erp/vore_sounds` is replaceable by existing `/datum/vore_pref/toggle/eating_noises`
`/datum/preference/toggle/cyborg_sleepers` would require its own vore pref

Check how these are used in 
https://github.com/SPLURT-Station/S.P.L.U.R.T-tg/blob/master/modular_zubbers/code/datums/components/vore/vore.dm

https://github.com/SPLURT-Station/S.P.L.U.R.T-tg/blob/master/modular_zubbers/code/datums/components/vore/preferences.dm

https://github.com/SPLURT-Station/S.P.L.U.R.T-tg/blob/master/modular_zubbers/code/datums/components/vore/preferences.dm